### ISSUE TITLE
Pulling image from private registry fails

### ISSUE THREAD

**Issue by vaidx** [lensesio/lenses-helm-charts#11]
When I try to pull the lenses.io image from my private registry, I receive the following message:

`Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: 
ValidationError(Deployment.spec.template.spec.containers[0]): unknown field "imagePullSecrets" in io.k8s.api.core.v1.Container`

Excerpt from `values.yaml` file:

```
image:
  repository: lensesio/lenses
  pullPolicy: IfNotPresent
  tag: 4.3.3
  registrySecretKey: regcred-ecr
```

I believe this is due to the incorrect placement of `imagePullSecrets` field in the `deployment.yaml` in Helm Chart.

```
      containers:
      - name: {{ .Chart.Name }}
        image: "{{ .Values.image.repository }}:{{if .Values.image.tag }}{{ .Values.image.tag }}{{else}}{{ .Chart.AppVersion }}{{end}}"
        imagePullPolicy: {{ .Values.image.pullPolicy }}
        {{- if .Values.image.registrySecretKey }}
        imagePullSecrets:
        - name: {{ .Values.image.registrySecretKey }}
        {{- end }}
```

**Comment by vaidx** [comment]
Issued a PR to fix this. Have a look, thx! (https://github.com/lensesio/lenses-helm-charts/pull/12)

**Comment by georgeyord** [comment]
Fix merged on version `4.3.13`, thanks again for the contribution! 🚀
